### DISCUSSION TITLE
URL shortener and shortmenu app

### DISCUSSION THREAD

**Discussion by ghost** [diced/zipline#461] [Questions] [answered]
Anyone have any examples for setting up the url shortener with the app shortmenu for Mac?

would be much appreciated, zipline was overall much more easy to get started with than shlink

**Comment by diced** [comment]
Don't think it's possible as Zipline's URL shortening works by sending a JSON post request to /api/shorten, and by the looks of it [here (shortmenu docs)](https://shortmenu.com/support/custom-services/), it doesn't let you do that. I may be wrong though, so I recommend looking into it (I don't own a mac or iphone).

**Comment by lerndmina** [comment, marked as answer]
Hey @gitekelund, I know this post is quite old, but on the off chance you've not found the solution, here it is.

Go to custom services and add one.
<img width="658" alt="image" src="https://github.com/diced/zipline/assets/10632157/29c357ec-352e-4aae-8b0b-fb69a642c07f">


Set the domain to your Zipline instance then follow the format here
<img width="437" alt="image" src="https://github.com/diced/zipline/assets/10632157/b67f644d-33da-4fd5-a505-d4aa2a3228dc">

I hope this helps :)
